controller/array_action/array_mediator_interface.py

```diff
@@ -321,6 +321,23 @@ def get_host_by_host_identifiers(self, initiators):
         """
         raise NotImplementedError
 
+    @abstractmethod
+    def get_host_by_name(self, host_name):
+        """
+        This function will find the host name by iscsi iqn or fc wwns.
+
+        Args:
+           host_name : name of the host in the storage system
+
+        Returns:
+           Host
+           None
```

Comment:
is this really necessary? I've never seen this before (used in addition to non-None)

Reply:
[here](https://github.com/IBM/ibm-block-csi-driver/blob/b0b5e7177f50aa76d72b7ff5290c6819b41a7da7/controller/array_action/array_mediator_interface.py#L226), there is always the first time..

Reply:
I don't like it, but if it's already used in the project...